app/models/activity.rb

```diff
@@ -3,8 +3,16 @@ class Activity
   include Mongoid::Timestamps
   include JudgeScoringHelper
 
+  # Github event actions for Issue can be any of assigned, unassigned, labeled, unlabeled, opened, edited, closed or reopened.
+  # Refer https://developer.github.com/v3/activity/events/types/#issuesevent for more details.
+  ISSUE_ACTIONS = %W(assigned unassigned labeled unlabeled opened edited closed reopened)
+
+  # Actions ignored for scoring.
+  IGNORED_FOR_SCORING = %W(closed)
```

Comment:
I think - the only actions we need to issue points for is "opened"  and "reopened" and nothing else. Otherwise, we leave the scope for users into tricking the system by re-assigning, changing labels and editing an issue to keep getting more points!